src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindowBase.cs

```diff
@@ -19,6 +19,15 @@ internal class CompositorWindowBase<TWindow> : EditorWindowBase<TWindow> where T
         protected const string AppDeviceTypeLabel = "App";
         protected string holographicCameraIPAddress;
 
+        private readonly List<Guid> unsupportedLocalizers = new List<Guid>

```

Comment:
>private readonly List<Guid> unsupportedLocalizers = new List<Guid> [](start = 8, length = 66)

Can we also have a comment here that explains more about the unsupported nature of this list? i.e. down below where it's used, we join two different list - both an unsupported and a supported one. A cursory reading leaves a reader somewhat confused because it seems like everything that is supported... should probably be all you need to check.

The other possibility is to rename things down below (i.e. not "supported" but "candidate" localizers).

Either way, having some more context about why this additional filtering is helpful.